css/popup.css

```diff
@@ -0,0 +1,138 @@
+div.popup-container {
+  display: flex;
+  justify-content: center;
+}
+
+div.popup-holder {
+  z-index: 2;
+  transform: scale(1);
+  transition: all 1s ease-in-out;
+  opacity: 1;
+}
+
+div.popup-holder.active {
+  z-index: -3;
+  transform: scale(0.2);
+  opacity: 0.2;
+}
+
+.project_info.popup {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+  min-height: 110vh;
+  background-color: #fff;
+  border-radius: 20px;
+  margin: 0.8rem auto;
+  padding: 20px;
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  max-width: 800px;
+  -webkit-box-shadow: -8px -4px 300px 0 rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: -8px -4px 300px 0 rgba(0, 0, 0, 0.75);
+  box-shadow: -8px -4px 300px 0 rgba(0, 0, 0, 0.75);
+}
+
+.project_info.popup.active {
+  z-index: 1;
+}
+
+.project_info .popup-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.project_info a {
+  text-decoration: none;
+  font-weight: 400;
+  font-size: 0.6rem;
+  border: 1px solid #6070ff;
+  border-radius: 5px;
+  padding: 5px 5px;
+}
```

Comment:
- [ ] Please know that the font size is very small, kindly follow the design guidelines provided.